contracts/lib/SafeMath.sol

```diff
@@ -1,6 +1,6 @@
-pragma solidity ^0.4.23;
+pragma solidity ^0.4.24;
```

Comment:
Other solidity files are using 0.4.23. Should we consider using 0.4.24 in all new files?

Reply:
Hm, I don't know. As solidity is developing relatively quickly including the addition of new features, I would say update whatever you work on? What do you think?

Reply:
We can keep `0.4.23`, and when we plan to upgrade we can create a ticket.